docs/userguide/quickstart.rst

```diff
@@ -101,7 +101,8 @@ The following example demonstrates a minimum configuration
         [options]
         install_requires =
             requests
-            importlib-metadata; python_version < "3.8"
+           importlib-metadata; python_version<"3.10"
```

Comment:
Is this an indentation inconsistency, or is it just github not rendering the diff correctly?

Reply:
Not a github issue (since it's using spaces not tabs). Thanks for noticing.
I copied the line and other sections are using 1 less space.